Week3/Recursive_exp/Recursive_exp/recursive_exp.cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+
+double exponention(double a, int x) {
+	if (x % 2 == 0)		 // even
+		return (x == 1) ? a : exponention(a * a, x / 2);
+	else				// odd
+		return (x == 1) ? a : a * exponention(a * a, (x-1)/2);
+}
```

Comment:
please use curly braces, PLEASE.
consider also proper revising, because currently exact types are not clear for me.